root_utils/root_file_utils.py

```diff
@@ -2,7 +2,7 @@
 import os
 import numpy as np
 
-ROOT.gSystem.Load(os.environ['WCSIMDIR'] + "/libWCSimRoot.so")
+ROOT.gSystem.Load(os.environ['WCSIM_BUILD_DIR'] + "/lib/libWCSimRoot.so")
```

Comment:
same as above

Reply:
I think this line is also no longer needed at all